src/main/java/pro/sky/java/school/data/Question.java

```diff
@@ -0,0 +1,43 @@
+package pro.sky.java.school.data;
+
+import java.util.Objects;
+
+public class Question {
+
+    private String question;
+    private String answer;
```

Comment:
```suggestion
    private final String question;
    private final String answer;
```
поскольку поля неизменяемы